internal/rest/dequeue_nodes.go

```diff
@@ -1306,7 +1306,7 @@ func (rest *RestOperations) VSVipCU(vsvip_nodes []*nodes.AviVSVIPNode, vs_cache_
 							utils.AviLog.Debugf("key: %s, msg: the checksums are same for VSVIP %s, not doing anything", key, vsvip_cache_obj.Name)
 						} else {
 							// The checksums are different, so it should be a PUT call.
-							restOp, err := rest.AviVsVipBuild(vsvip, vsvip_cache_obj, key)
+							restOp, err := rest.AviVsVipBuild(vsvip, vs_cache_obj, vsvip_cache_obj, key)
```

Comment:
In case we do get an error, the vsvip operation would be skipped, but do we want to proceed with other operation ?... e.g. pool also has a rhi enabled flag.